.github/workflows/deploy.yml

```diff
@@ -74,9 +62,26 @@ jobs:
         if: matrix.distribution == 'bdist_wheel'
         run: |
           pip install --upgrade setuptools wheel
-          python setup.py ${{ matrix.distribution }} --python-tag ${{ matrix.wheel-tag}}
-      - name: Publish
-        uses: pypa/gh-action-pypi-publish@v1.0.0a0
+          python setup.py ${{ matrix.distribution }} --python-tag 3${{ matrix.python-3-version}}
+          
+      - name: Upload artifacts
+        uses: actions/upload-artifact@v2
         with:
-          user: __token__
-          password: ${{ secrets.PYPI_PWD }}
+          path: dist/*
+          
+  upload_all:
```

Comment:
I imagine we can probably create the sdist in this job right before we upload to pypi.

Reply:
Yes, I usually make it in a lone job (since this job for me usually only runs on tags, while the previous jobs can run other places), but here is fine.

I though I `include:`d the SDist?

Reply:
Ah, I see in your other comment, thanks!